source/jormungandr/tests/end_point_tests.py

```diff
@@ -206,3 +206,20 @@ def test_lines_context(self):
         response = self.query('/v1/coverage/main_routing_test/lines', display=True)
         self.check_context(response)
         assert response['context']['timezone'] == 'UTC'
+
+    def test_equipment_reports_context(self):
+        # TODO : When Kraken API is ready, replace
```

Comment:
Maybe you want to mention [#NAVP-1220](https://jira.kisio.org/browse/NAVP-1220) here ? Or update the ticket with this TODO.

Reply:
I'll update the ticket with this TODO :heart_decoration: